src/java.base/share/classes/java/lang/Character.java

```diff
@@ -169,6 +169,7 @@
  * use instances for synchronization, or unpredictable behavior may
  * occur. For example, in a future release, synchronization may fail.
  *
+ * @spec https://www.unicode.org/reports/tr27 Unicode 3.1.0
```

Comment:
This should probably be removed, as the original link (explaining `U+n` notation) is broken.

Reply:
@naotoj The edits are driven by a script, using info about existing links in the same doc comment. If you don't think this reference is appropriate, it would be better to either remove the existing link (and I'll regenerate this patch) or else this patch goes through and you fix up both the existing link and the `@spec` tag afterwards.

Reply:
Either way is fine with me. I will fix it up if you choose the latter.